src/sagemaker_sklearn_extension/contrib/taei/README.md

```diff
@@ -0,0 +1,137 @@
+# Tabular AutoEncoder Interpolator
+
+## Overview
+<img src="images/overview.png" alt="overview" height="200">
+
+This library contains implementations of the latent space minority oversampling techniques proposed in [1] for 
+multi-modal data. These oversamplers work by
+
+1. Mapping the multi-modal samples to a dense continuous latent space using an autoencoder
+2. Applying oversampling by interpolation in the latent space
+3. Mapping the synthetic samples back to the original feature space
+
+This framework was shown to be effective in generating high-quality multi-modal synthetic data which then resulted in 
+better prediction quality for downstream tasks.
+
+#### LatentSpaceOversampler
+The interpolator is implemented by `LatentSpaceOversampler` which takes two inputs:
+- `model` - The autoencoder used to map the samples to the latent space and back. Currently, two 
+autoencoders are provided with the package: `AE` which is a vanilla autoencoder and `VAE` which is a variational 
+autoencoder.
+- `base_oversampler` (function) - The oversampling function applied in the latent space. No oversamplers are provided 
+with this package. We have experimented with `SMOTE` from 
```

Comment:
We now have the `StarOversampler` right? `README` should be updated.

Reply:
There is some documentation. See https://github.com/YotamElor/sagemaker-scikit-learn-extension/tree/master/src/sagemaker_sklearn_extension/contrib/taei#variational-autoencoder--polynomfit